index.html

```diff
@@ -784,14 +783,15 @@ <h4>Legal</h4>
                                             <li><a href="#">Media</a></li>
                                         </ul>
                                     </div>
-                                </div>
+                                </div> -->
                             </div>
                         </div>
-                    </div> -->
+                    </div>
+                    <br><br><br>
                     <div class="row" style="margin-bottom: 30px;">
                         <div class="col-md-12">
                             <div class="footer-copyright center">
-                                <p>&copy;2021 DBIT SevaTech.</p>
+                                <p>&copy;2023 DBIT SevaTech.</p>
```

Comment:
you can keep this as a dynamic code, which gets the current year. This would skip updaitng the year every time. next month you'll have to update again.